docs/en/integrations/data-ingestion/clickpipes/kafka.md

```diff
@@ -193,6 +197,24 @@ For Apache Kafka protocol data sources, ClickPipes supports [SASL/PLAIN](https:/
 ### IAM
 AWS MSK authentication currently only supports [SASL/SCRAM-SHA-512](https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html) authentication.
 
+## Performance
+
+### Batching
+ClickPipes inserts data into ClickHouse in batches. This is to avoid creating too many parts in the database which can lead to performance issues in the cluster.
+
+Batches are inserted when one of the following criteria has been met:
+- The batch size has reached the maximum size (100,000 rows or 20MG)
```

Comment:
Everything else looks good but this one thing is standing out to me: `20MG`

I assume it's megabyte, but I usually see it written as one of the following: `MB`, `mb`, `Mb`, `MiB` (some more specific depending on whether they mean standard 1024 or 1000). My brain keeps reading milligram 🧑‍🔬